doc/examples.md

```diff
@@ -118,30 +74,28 @@ In many cases it is useful to write files to the root filesystem. This example w
 }
 ```
 
-The config makes use of the universally-defined "root" filesystem. This filesystem is defined within Ignition itself and roughly looks like the following. The "root" filesystem allows additional configs to reference the root filesystem, regardless of its type (e.g. btrfs, tmpfs, ext4).
+Paths are specified relative to the root filesystem of the final system, not the initramfs Ignition runs in. Symlinks are followed as if Ignition was running from the final system, not the initramfs. See the [operator notes] for more information about how Ignition follows symlinks.
```

Comment:
Let's try to avoid talking about the initramfs.  I think that's an implementation detail users shouldn't have to think about.